examples/minimal-static-router/src/main.rs

```diff
@@ -160,19 +161,20 @@ impl LinkBuilder<EthernetFrame, EthernetFrame> for Router {
                     }))
                     .build_link();
             all_runnables.append(&mut ipv6_subnet_router_runnables);
+            AutoEgressor!(ipv6_subnet_router_egressors, sub_a, sub_b, sub_c);
```

Comment:
It's cool that this macro can extract each of the `Vec`'s values into different variables. The only nits I have are 

1. The macro itself doesn't have anything to do with egressors. It just unpacks `Vec`s. Maybe `vecUnpack!`?
2. If you didn't supply the right number of values we would get weird behavior, like trying to remove from an empty `Vec` I imagine. This version will probably panic at runtime while the router is being initialized, but it would be nice if we could verify the number of variables matches the length of the vec earlier.